.github/workflows/main.yml

```diff
@@ -15,39 +15,54 @@ jobs:
           - os: macos-10.15
             compiler: gfortran-10
 
-      # fail-fast is set to 'true' here which is good for production, but when
-      # debugging, set to 'false'. fail-fast means if *any* ci test in the matrix fails
-      # GitHub Actions will stop any other test immediately. So good for production, bad
-      # when trying to figure something out. For more info see:
-      # https://www.edwardthomson.com/blog/github_actions_6_fail_fast_matrix_workflows.html
-
-      fail-fast: true
+      fail-fast: false
     env:
       FC: ${{ matrix.compiler }}
       LANGUAGE: en_US.UTF-8
       LC_ALL: en_US.UTF-8
       LANG: en_US.UTF-8
       LC_TYPE: en_US.UTF-8
-      OMPI_ALLOW_RUN_AS_ROOT: 1
-      OMPI_ALLOW_RUN_AS_ROOT_CONFIRM: 1
-      OMPI_MCA_btl_vader_single_copy_mechanism: none
 
     name: ${{ matrix.os }} / ${{ matrix.compiler }}
     steps:
-      - name: Compiler Versions
+      - name: Versions
         run: |
           ${FC} --version
           cmake --version
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 1
-      - name: Build GFE Prereqs
```

Comment:
Is there a reason you removed this? I added this because at some point @tclune and I might try and "fix" the oddness that is happening with the submodule externs in GFE stack (where even if gFTL exists, it builds the external ones).

Reply:
Because gftl is built by default by cmake as a submodule, no need to build it before gftl-shared.